README.md

```diff
@@ -20,25 +20,26 @@ make test
 
 ## Testing locally
 
-1. install nftables_exporter: `go install github.com/Sheridan/nftables_exporter`
-
 ```bash
 # start kind cluster
 kind create cluster
 
-# start exporters
-node_exporter &
-nftables_exporter --config ./nftables_exporter.yaml &
-
 # deploy manifests
 k apply -f deploy
 
 # start the controller
 bin/firewall-controller --hosts-file ./hosts
 
 # watch results
-k describe networktraffic
 k describe -n firewall firewall
 cat nftables.v4
 cat hosts
 ```
+
+## Suricata
+
+By default only basic statistics are reported via the firewall crd, but id ids is enabled all events can be forwarded to a specified destination.
```

Comment:
```suggestion
By default only basic statistics are reported via the firewall crd, but if ids is enabled all events can be forwarded to a specified destination.
```

Reply:
Yes this is the plan, the basic metrics can be gather via unix-socket as well.